author john Dey Fred Hutch Cancer Center
fix rle.h:33 multiple definition of `rle_auxtab'
Binary files NextPolish-cfa7b1b1de93a46d2a24195ba1a7732631d35bee.orig/source/.Makefile.swp and NextPolish-cfa7b1b1de93a46d2a24195ba1a7732631d35bee/source/.Makefile.swp differ
diff -ruN NextPolish-cfa7b1b1de93a46d2a24195ba1a7732631d35bee.orig/source/util/bwa/rle.h NextPolish-cfa7b1b1de93a46d2a24195ba1a7732631d35bee/source/util/bwa/rle.h
--- NextPolish-cfa7b1b1de93a46d2a24195ba1a7732631d35bee.orig/source/util/bwa/rle.h	2025-01-09 01:18:03.000000000 +0000
+++ NextPolish-cfa7b1b1de93a46d2a24195ba1a7732631d35bee/source/util/bwa/rle.h	2025-09-11 22:03:55.341695000 +0000
@@ -30,7 +30,7 @@
  *** 43+3 codec ***
  ******************/
 
-const uint8_t rle_auxtab[8];
+#  const uint8_t rle_auxtab[8];
 
 #define RLE_MIN_SPACE 18
 #define rle_nptr(block) ((uint16_t*)(block))
